assets/src/domain/blocks/services/apollo/queries/useAttendees.ts

```diff
@@ -0,0 +1,33 @@
+import { QueryResult } from '@apollo/react-common';
+import gql from 'graphql-tag';
+
+import { AttendeesQueryWhereArgs, useAttendeesQuery } from '@dataServices/apollo/queries';
+import { AttendeesList } from '@blocksServices/apollo/types';
+
+export const GET_ATTENDEES: any = gql`
+	query GET_ATTENDEES($first: Int, $where: EspressoRootQueryAttendeesConnectionWhereArgs) {
+		espressoAttendees(first: $first, where: $where) {
+			nodes {
+				id
+				dbId
+				avatar
+				fullName
+			}
+		}
+	}
+`;
+
+type UseAttendees = (whereArgs: AttendeesQueryWhereArgs, limit?: number) => QueryResult<AttendeesList>;
+
+const useAttendees: UseAttendees = (where, limit) => {
+	return useAttendeesQuery({
+		query: GET_ATTENDEES,
+		variables: {
+			first: limit,
+			where,
+		},
+		fetchPolicy: 'cache-first',
```

Comment:
is this overridden somewhere, because I see that default is `cache-first`?

![image](https://user-images.githubusercontent.com/1477453/81278432-57a55500-905e-11ea-8f49-270d5a35d820.png)

Reply:
Yes, in `useCacheQuery` we set it to `cache-only` by default.